src/test/java/seedu/address/testutil/TaskBuilder.java

```diff
@@ -82,8 +83,14 @@ public TaskBuilder withStatus(Boolean isComplete) {
         return this;
     }
 
+    /**
+     * Build a Task based on the fields specified.
+     * @return The Task object built.
+     */
     public Task build() {
-        return new Task(moduleName, taskId, taskName, taskDeadline);
+        return this.isComplete
+                ? new Task(moduleName, taskId, taskName, taskDeadline, isComplete)
+                : new Task(moduleName, taskId, taskName, taskDeadline);
```

Comment:
Is this the same as return `new Task(moduleName, taskId, taskName, taskDeadline, this.isComplete)` ?

Reply:
Yes i realise they are the same. thanks!